docs/developer/HootenannyDevelopmentEnvironmentSetupInstructions.asciidoc

```diff
@@ -1,756 +1,20 @@
 
 == Hootenanny Development Environment Setup Instructions
 
-This document contains the current set of instructions to install the Hootenanny developer build environment within a VirtualBox virtual machine running Ubuntu 14.04.  An alternative more streamlined approach to installing Hootenanny can be taken using the Vagrant file instructions provided in <<HootenannyVagrantSetup>>.  If you do not want to work with VirtualBox or Vagrant, an unsupported script to install to bare metal summarizing the contents of this document is located at: $HOOT_HOME/scripts/InstallHootDevelopmentEnvironment.sh
+You have two options.
 
-=== External Dependencies
+Option #1:
 
-These steps often require additional assistance so it's best to complete these first.
+Use a Vagrant VM to setup the Hootenanny application without any development tools, and either install your custom development tools on the VM or use development tools already installed on your local machine to interact with the VM.  See the Vagrant VM setup instructions provided at: https://github.com/ngageoint/hootenanny/blob/master/VAGRANT.md for details.
 
-=== Update Host OS
+Option #2:
 
-. `sudo apt-get update`
-. `sudo apt-get upgrade`
-. `sudo apt-get dist-upgrade`
-. `sudo reboot`
+If you want to build a development environment completely on bare metal for the highest possible performance:
 
-=== Install VirtualBox
+* Download https://github.com/ngageoint/hootenanny/blob/master/scripts/InstallHootDevelopmentEnvironment.sh to your home directory.
+* Edit your home path in the script and run it.  Hoot will be cloned and an (almost) complete development environment will be set up from scratch.  
+* If you use the installed Eclipse IDE for services Java development, there are some additional configuration steps you will need to complete as described here: https://127.0.0.1:9443/redmine/projects/hootenany/wiki/Developer_-_Install_IDEs
```

Comment:
Oh, that's how I tunnel into Redmine (copy/paste)...so would only work for someone tunneled and configured the same as me...not too helpful.  Redmine isn't available, afaik, without tunneling so not sure what else could be put there now.  I guess we probably shouldn't reference Redmine at all in these documents that are going into the gihub repo, right?